### DISCUSSION TITLE
Case of no ionization and z < 1

### DISCUSSION THREAD

**Discussion by DanRRRR** [epochpic/epoch#486] [General]
All the demo examples without ionization (z=const) imply that the initial charge is integer number and the const > 1 (case with ionization we are not considering now). Is it possible to set the case when this const is < 1, say, z=0.01 ?

**Comment by Status-Mirror** [comment]
The `charge` parameter in the EPOCH input deck species block refers to the charge of a real particle belonging to that species, specified in units of the elementary charge. It does not have to be an integer.

Just remember that ions and electrons all have a charge which may be expressed as an integer number of elementary charges. Some of the physics packages will expect you to use real particles, and will assume you _do_ have an integer number of elementary charges. The behaviour of these packages will be unpredictable if you give them particles which aren't ions, electrons, positrons or photons.

For more information, check the documentation on the [species block](https://epochpic.github.io/documentation/input_deck/input_deck_species.html)
